### ISSUE TITLE
GridColumnMenuFilter scrolls window if certain specific html/body/React root CSS doesn't exist

### ISSUE THREAD

**Issue by mattfelten** [telerik/kendo-react#758]
## I'm submitting a...
Bug report

## Current behavior

With specific CSS on `html`, `body`, and `my-app`, clicking the filter icon in a Grid opens a dropdown as expected.
Without that CSS, clicking the filter will scroll the page.

## Expected behavior

I would expect the window to not scroll.

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/react-mp9fzg?file=app%2Fmain.jsx
1. remove the CSS from index.html
1. scroll window so table is visually at the top of the window
1. click the filter icon so the dropdown appears
1. watch the window scroll.
1. bring back the CSS
1. click the filter icon again
1. the window doesn't scroll

## Environment

Package versions:
See StackBlitz. This was happening down to 3.14 as well (which is where I found it), but is still happening with latest.

Browser:

- Chrome Latest
- Safari Latest

System:
- TypeScript version: XX 
- Node version: XX  
- Platform:  MacOS

**Comment by yakunins** [comment]
@mattfelten The problem may be fixed by replacing `html-body-myapp` styles with these, effectively disabling FilterMenu up and down animation:

```
		.k-slide-down-enter {
			transform: translateY(0%);
			opacity: 0;
		}

		.k-slide-down-enter-active {
			transform: translateY(0%);
			opacity: 1;
		}

		.k-slide-down-exit-active {
			transform: translateY(0%);
			opacity: 1;
		}

		.k-slide-down-exit {
			transform: translateY(0%);
			opacity: 0;
		}
```

https://stackblitz.com/edit/react-mp9fzg-md7mbm?file=index.html

**Comment by mattfelten** [comment]
From your example, it seems like only the `opacity` change is needed to prevent that extra scrolling. https://stackblitz.com/edit/react-mp9fzg-r8zr9a?file=index.html

**Comment by yakunins** [comment]
No, to prevent scrolling `transform: translateY(100%)` need to be replaced with `transform: translateY(1%)` on animation start and end. Or a combination `opacity: 0` + `transform: translateY(0%)`. Because at least one css property change is requered — as react tracks styles of animated node and apply proper classes subsequntly. 

There is a three main variants of show/hide аnimation:

1. `transform: translateY(100%→0%)` is a default, but triggers page scroll
2. `transform: translateY(1%→0%)` is an easy solution
3. `transform: translateY(0%→0%), opacity: 0 → 1` is a better solution

See https://stackblitz.com/edit/react-mp9fzg-fzzney?file=index.html

**Comment by PaskalevStoyan** [comment]
Closing this issue as it is not reproducible in the latest version of the KendoReact Grid - `v9.3.0`.
